app/services/user-agent-service/datomstore.js

```diff
@@ -110,33 +152,45 @@ export class ProfileDatomStorage {
     return true;
   }
 
+  getDB() {
+    return datascript.core.db(this.conn);
+  }
+
+  transact(assertions) {
+    return datascript.core.transact_BANG_(this.conn, assertions);
+  }
+
   async startSession(scope, ancestor, now = microtime.now(), reason = SessionStartReason.newTab) {
-    const result = datascript.core.transact_BANG_(this.conn, vector(
+    let assertions = vector(
       vector(DB_ADD, -1, 'session/startedInScope', `${scope}`),
```

Comment:
At one point our codebase was doing both strings (e.g., 'sync') and integers (generated by sqlite!). I settled on strings.